src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,22 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] machineList = new Machine[]{
+                truck,
+                bulldozer,
+                excavator
+        };
 
+        for (Machine machine : machineList) {
+            machine.doWork();
+        }
+
+        for (Machine machine : machineList) {
+            machine.stopWork();
+        }
```

Comment:
redundant split single loop logic to the 2 loops. think about how to combine them in single loop